src/Components/Navbar.js

```diff
@@ -23,6 +23,10 @@ export default function Navbar()
         <div className="flex lg:flex-1 text-lg font-semibold">
           <Link to={"/"}><img src='/Images/jira-low-resolution-logo-color-on-transparent-background.png' alt='PMT LOGO' height="30px" width="30px"></img></Link>
         </div>
+        {/* <legend style={linkStyle} className="text-lg font-semibold leading-6 text-gray-900 lg:justify-end pointer-cursor" onClick={handleLogout}>
+            Logout <span aria-hidden="true">&rarr;</span> 
```

Comment:
Please remove this commented part if not required in future

Reply:
i have done all the required changes in that file

Reply:
Ok